src/displayWeather.js

```diff
@@ -0,0 +1,30 @@
+import fetchImage from './fetchImage';
+import fetchData from './fetchData';
+import degreeUnit from './degree';
+import error from './error';
+
+const display = document.getElementById('display');
+
+const displayWeather = async () => {
+  const { cod, main, weather, unit, name } = await fetchData();
+  switch (cod) {
+    case 200: display.innerHTML = '';
+      display.innerHTML += `
+    <div class="infoHold">
+    <span id="1">City: ${name}</span>
```

Comment:
won't it be nicer if we had this UI updated in a dedicated method?
[maybe/Suggestion]: status good  call a ui method and pass the data to it.
else call another method and tell it the problem.

**Note:**
implement as you like but separate concerns and let the error tell me whats wrong.